src/main/java/seedu/address/model/loan/Loan.java

```diff
@@ -60,6 +84,23 @@ public Address getAddress() {
         return Collections.unmodifiableSet(tags);
     }
 
+    /**
+     * Change the loan status to the newStatus as provided.
+     * Throws SameLoanStatusException if the newStatus is the same as the previous status.
+     * @param newStatus
+     * @return true if the function managed to complete.
+     * @throws SameLoanStatusException
+     */
+    public boolean changeLoanStatus(LoanStatus newStatus) throws SameLoanStatusException {
```

Comment:
Since `SameLoanStatusException` is a RunTimeException, I don't think you need to declare it in the method signature.

Specifying it in the documentation (and when it gets thrown) is still a good idea, however.